### PR TITLE
doc/SubmittingPatches-backports.rst: clarify backport PR labels

### PR DESCRIPTION
**Pull request by smithfarm**

### CODE REVIEW

**Comment by tchaikov** [comment]
jenkins test docs

File: SubmittingPatches-backports.rst [hunk 1]
```diff
@@ -334,14 +334,19 @@ Once the backport PR is open, the first order of business is to set the
...
-a comment to the PR with the following content::
-
-    @ceph/backport-admins
-
-This will alert the `Stable Releases and Backports team`_ to your PR, and
-someone will ensure your PR gets the proper labels.
-
+If you don't have sufficient GitHub permissions to set the Milestone, don't
+worry. Members of the `Stable Releases and Backports team`_ periodically run
+a script (``ceph-backport.sh --milestones``) which scans all PRs targetting stable
```
Comment (callithea) [inline SubmittingPatches-backports.rst:339]: s/targetting/targeting/

Comment (smithfarm) [reply SubmittingPatches-backports.rst:339]: yeah, I saw that, but figured we could just live with it... sorry

Comment (callithea) [reply SubmittingPatches-backports.rst:339]: ack, no worries!

**Pull request merged**